pkg/auth/providerrefresh/refresh_daemon.go

```diff
@@ -187,7 +209,12 @@ func (r *refresher) triggerUserRefresh(userName string, force bool) {
 	} else {
 		_, err = r.userAttributes.Update(attribs)
 		if err != nil {
-			logrus.Errorf("Error updating user attribute to trigger refresh: %v", err)
+			if apierrors.IsConflict(err) {
+				// User attribute has just been updated, triggering the refresh.
+				logrus.Debugf("Error updating user attribute to trigger refresh: %v", err)
+			} else {
+				logrus.Errorf("Error updating user attribute to trigger refresh: %v", err)
```

Comment:
just to confirm that we don't want to process the error, and fine with just logging it

Reply:
yes. Not much more can be done and the next api request would retrigger the logic.